### DISCUSSION TITLE
Request for new Feature - Notes fields in budgets

### DISCUSSION THREAD

**Discussion by GabrielBarcia** [firefly-iii/firefly-iii#7764] [General]
Hello,
Could it be possible to add a notes field in the budget screen?

The use case is that if you have a budget that is a umbrella of several items, then can write in the description the logic so is easier to edit and remember in the future.

Thanks
Gabriel

**Comment by JC5** [comment]
I think it's on the list already somewhere. :+1:

**Reply by GabrielBarcia** [reply]
Excelent then. Thanks
